src/rp2_common/pico_runtime/runtime.c

```diff
@@ -238,7 +238,7 @@ __attribute((weak)) int _times(struct tms *tms) {
 #if CLOCKS_PER_SEC >= 1000000
     tms->tms_utime = to_us_since_boot(get_absolute_time()) * (CLOCKS_PER_SEC / 1000000);
 #else
-    tms->tms_utime = to_us_since_boot(get_absolute_time()) / (1000000 / CLOCKS_PER_SEC);
+    tms->tms_utime = (clock_t)to_us_since_boot(get_absolute_time()) / (1000000 / CLOCKS_PER_SEC);
```

Comment:
Do you need to cast the call to to_us_since_boot above as well?

Reply:
yes; i've picked up your fix now